vmdb/app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -12,7 +12,7 @@ def save_inventory_multi(type, parent, hashes, deletes, find_key, child_keys = [
     # Delete the items no longer found
     unless deletes.blank?
       $log.info("MIQ(#{self.name}.save_#{type}_inventory) Deleting #{self.log_format_deletes(deletes)}")
-      parent.send(type).delete(deletes)
+      parent.send(type).delete(deletes - new_records)
```

Comment:
@tenderlove I don't understand why this works for you.  The deletes, at this point, contains the set of records to delete and will not have new_records in it.  So, basically, I think this is just doing `deletes - [records that don't matter]` ~> `deletes.dup`?  Even so, I don't get why that matters?